unison-core/src/Unison/HashQualified.hs

```diff
@@ -49,10 +49,9 @@ toName = \case
 --   [.foo.bar, foo.bar] -> [foo.bar, .foo.bar]
 sortByLength :: [HashQualified Name] -> [HashQualified Name]
 sortByLength hs = sortOn f hs where
-  f (NameOnly n) = (countDots n, 0, Left n)
```

Comment:
No behavior change intended here, just noticed we were round-tripping through `Text` to count the segments in a name, which is no longer necessary.